src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,86 @@
+//dependencie imports
```

Comment:
We should use eslint-plugin-simple-import-sort to automate this I think.

https://www.npmjs.com/package/eslint-plugin-simple-import-sort

Reply:
I think i got this set up, but im not 100% sure if its sorting.